app/assets/stylesheets/all/sequencescape.scss

```diff
@@ -1364,39 +1364,6 @@ td.orange {
   border: 0;
 }
 
-div.characterisation {
```

Comment:
These styles were exclusively associated with the old pipeline. (Ahh, the joys of distinct class names)